README.Rmd

```diff
@@ -57,4 +57,38 @@ data(study)
 1. Categorical with many different categories, for example the `id` column of the `study` data frame.     
 2. Categorical with few different categories, for example the `gender` column of the `study` data frame.     
 3. Date, for example the `date` column of the `study` data frame.    
-4. Numeric, for example the `height` column of the `study` data frame.    
```

Comment:
Why is all of this showing up as new? This text is already in the README.

Reply:
I'm sorry, I don't know why. I did not change any of the text that is already there.

Reply:
Thank you!